docs/programming-guide.md

```diff
@@ -237,9 +237,11 @@ You can customize the `ipython` command by setting `PYSPARK_DRIVER_PYTHON_OPTS`.
 the [IPython Notebook](http://ipython.org/notebook.html) with PyLab plot support:
 
 {% highlight bash %}
-$ PYSPARK_DRIVER_PYTHON=ipython PYSPARK_DRIVER_PYTHON_OPTS="notebook --pylab inline" ./bin/pyspark
+$ PYSPARK_DRIVER_PYTHON=ipython PYSPARK_DRIVER_PYTHON_OPTS="notebook" ./bin/pyspark
 {% endhighlight %}
 
+After IPython Notebook server is launched, you can create a new "Python 2" notebook from "Files" tab. Inside the notebook, you can input the '%pylab inline' command as part of your notebook before you start to try spark from IPython notebook.
```

Comment:
Just out of curiosity, it's not supported as of what version -- something old, or recent?
Nit: you can back-tick-quote the `%pylab inline` bit.
Is it worth explaining what this option does, then? (Could be entirely obvious, just asking.)